Formula/vineyard.rb

```diff
@@ -0,0 +1,104 @@
+class Vineyard < Formula
+  include Language::Python::Virtualenv
+
+  desc "In-memory immutable data manager. (Project under CNCF)"
+  homepage "https://v6d.io"
+  url "https://github.com/v6d-io/v6d/releases/download/v0.6.0/v6d-0.6.0.tar.gz"
+  sha256 "26a024bd143a19f1ba9fb87fa9eb9fec53f19836d8214b35bc472fa2132033c3"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+  depends_on "apache-arrow"
+  depends_on "boost"
+  depends_on "etcd"
+  depends_on "etcd-cpp-apiv3"
+  depends_on "gflags"
+  depends_on "glog"
+  depends_on "libgrape-lite"
+  depends_on "llvm"
+  depends_on "nlohmann-json"
+  depends_on "open-mpi"
+  depends_on "openssl@1.1"
+  depends_on "tbb"
+
+  fails_with gcc: "5"
+
+  def install
+    # make libclang.so/libclang.dll findable by clang.cindex.Index.create()
+    ENV["LD_LIBRARY_PATH"] = "#{ENV["LD_LIBRARY_PATH"]}:#{Formula["llvm"].opt_lib}"
```

Comment:
This doesn't work on macOS.

Reply:
Could I restore to use the resource `libclang`?

Actually the pypi package is a slightly modified version where a variable `LIBCLANG_LIBRARY_PATH` is used to guide the search of the shared library.

On Mac the `cdll.LoadLibrary` seems only consider some hard-coded target directory: https://docs.python.org/3/library/ctypes.html#finding-shared-libraries

Reply:
That's not necessary. Try this instead:

```suggestion
    ENV.prepend_path "LD_LIBRARY_PATH", Formula["llvm"].opt_lib
    ENV.prepend_path "DYLD_FALLBACK_LIBRARY_PATH", Formula["llvm"].opt_lib
```

Reply:
The `DYLD_FALLBACK_LIBRARY_PATH` doesn't work as well: https://github.com/Homebrew/homebrew-core/runs/7161180593?check_suite_focus=true